update.sh

```diff
@@ -58,67 +59,67 @@ for repo in "${repos[@]}"; do
 	if [ -x "$repo/update.sh" ]; then
 		( set -x; "$repo/update.sh" )
 	fi
-	
+
```

Comment:
I can, although I'd still like to have a commit that strips trailing whitespace merged in at some point.